blockchain/interfaces/ethereum_mock.go

```diff
@@ -1,291 +0,0 @@
-package interfaces
```

Comment:
Is this file not necessary anymore?

Reply:
nope replaced with the new mocking funcitonality